admin_pages/registrations/templates/registration_page_registration_questions.template.php

```diff
@@ -44,18 +44,18 @@
                                     EEH_Template::format_currency($item['ticket']->price(), false, false)
                                 );
                                 $qty_price = $item['ticket']->price() * $ticket_count[ $item['ticket']->ID() ];
-                                echo $item['ticket']->qty()
+                                echo ($item['ticket']->qty()
                                     ? ' &nbsp; x &nbsp; '
                                       . absint($ticket_count[ $item['ticket']->ID() ])
                                       . esc_html__(' tickets', 'event_espresso')
                                       . ' &nbsp; = &nbsp; '
                                       . EEH_Template::format_currency($qty_price)
```

Comment:
this could use one of those magic `sprintf()` fixes...

at least put `$ticket_count` and " tickets" within it (plz and thnx)